docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.ts

```diff
@@ -21,7 +22,7 @@ export class IconPreviewComponent implements OnInit {
     @ViewChild(PopoverDirective, { static: true }) popover: PopoverDirective;
 
     ngOnInit(): void {
-        this.uxComponentSnippet = `<ux-icon name="${this.name}"></ux-icon>`;
+        this.uxComponentSnippet = `<ux-icon name="${this.name}" size="${this.iconSizeValue}"></ux-icon>`;
 
```

Comment:
There also seems to be an issue, the size in the code doesn't seem to get updated:

[![Image from Gyazo](https://i.gyazo.com/c15719471d329ba127378b900d214bee.gif)](https://gyazo.com/c15719471d329ba127378b900d214bee)